### ISSUE TITLE
Module Update: Password Policy

### ISSUE THREAD

**Issue by aleayr** [govCMS/GovCMS7#438]
Available version: 7.x-1.12
Current version: 7.x-1.11

> ## Changes since 7.x-1.11
> #2469681 by AohRveTPV: Expiration warnings may not be sent if cron not run daily
> #2566535 by AohRveTPV: Duplicate update functions
> #2468449 by AohRveTPV: Cannot unblock reblocked expired account
> #2464895 by AohRveTPV: Rename "complexity" constraint
> #2562449 by AohRveTPV, mikebarkas: Saving the user node edit form may not unblock user
> #2455881 by AohRveTPV: Improve README.txt
> #2476113 by AohRveTPV: Exempting uid 1 from expiration is insecure default
> #2543798 by AohRveTPV: Make test names and descriptions more consistent
> #2527824 by AohRveTPV: No tests for "Admin (UID=1) password expires." setting
> #2490548 by AohRveTPV: In history constraint validation $account object is modified
> #2464779 by AohRveTPV: 7.x-1.x using D6-style tokens
> #2461665 by AohRveTPV: Add tests for expiration e-mails
> #2467941 by AohRveTPV: Mock time retrievals for testing
> #1332000 by AohRveTPV, erikwebb, rooby, slip, hanoii: Feature to exclude pages from force password change
> #2464835 by AohRveTPV: password_policy.js unused
> #2475553 by AohRveTPV: Add type hinting
> #2462647 by AohRveTPV: Two password history entries created for new users, one invalid
> #2464881 by AohRveTPV: Replace time() with REQUEST_TIME where possible
> #2464503 by sveldkamp: Complexity error message is misleading/confusing
> #2463527 by AohRveTPV: No tests for forced password changes using Password Change Tab
> Made various code improvements that should not affect behavior.

**Comment by tobybellwood** [comment]
closing in favour of #529
